core/lib/brightcontent/view_lookup/filter_field.rb

```diff
@@ -27,16 +31,26 @@ def field_name
         end
       end
 
+      def input_name
+        [field_name.to_s, predicate].reject(&:!).join("_")
```

Comment:
Yeah maybe your right, I thought using the `!`-method in this way was pretty cool, but it isnt very readable, I agree.